pkg/images/enricher/enricher_impl_test.go

```diff
@@ -1414,6 +1415,6 @@ func newEnricher(set *mocks.MockSet, mockReporter *reporterMocks.MockReporter) I
 		mockReporter, emptySignatureIntegrationGetter, nil)
 }
 
-func newCache() expiringcache.Cache {
-	return expiringcache.NewExpiringCache(1 * time.Minute)
+func newCache() cache.ImageMetadata {
+	return cache.ImageMetadata(expiringcache.NewExpiringCache(1 * time.Minute))
```

Comment:
There's no `expiringcache.UpdateExpirationOnGets` option here. This makes this test cache different to the real singleton case. Is it on purpose?

Reply:
IMHO, this can be tracked as a follow-up.
This PR was about changing the direct use of `expiringcache.Cache` to a more strongly typed cache. This is what is done here, and this particular piece of the change does not seem to alter the test behaviour.